game-app/game-core/src/main/java/games/strategy/triplea/attachments/UnitAttachment.java

```diff
@@ -642,13 +625,8 @@ private void setIsSub(final String s) {
   @VisibleForTesting
   public void setIsSub(final Boolean s) {
```

Comment:
Isn't the parameter kind of redundant now?

Reply:
Same as for `setIsSuicide()`, this was a bug. Fixed by setting `isSub` to `s`. I checked all maps and none set this to false, so it should be safe to fix.